src/lib/pages/backoffice/Acquisition/Vendor/VendorDetails/VendorInformation.js

```diff
@@ -4,17 +4,21 @@ import { EmailLink } from '@components/EmailLink';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { Grid } from 'semantic-ui-react';
+import { invenioConfig } from '@config';
 
 export class VendorInformation extends React.Component {
   render() {
     const { vendor } = this.props;
+
+    const emailBody = `${vendor.name}, \n\n\n ${invenioConfig.APP.emailFooter}`;
```

Comment:
Would it make sense to have a default email body template in `defaultConfig`? It can also be a function sth like

```
defaultEmailBody: name => `${name} Kind regards,\nInvenioILS`
```
and we can remove the email footer emailFooter.